src/App.tsx

```diff
@@ -1,61 +1,153 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { User } from './components/types/User';
+import { TodoWithUser } from './components/types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
+
+const preparedTodos: TodoWithUser[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: usersFromServer.find(({ id }) => id === todo.userId) || null,
+} as TodoWithUser));
```

Comment:
Avoid using `as`, in this case, you already declared that `preparedTodos` have `TodoWithUser[]` type